src/mudlet-lua/lua/geyser/GeyserReposition.lua

```diff
@@ -13,3 +13,4 @@ function GeyserReposition()
 end
 
 registerAnonymousEventHandler("sysWindowResizeEvent", "GeyserReposition")
+registerAnonymousEventHandler("sysUserWindowResizeEvent", "GeyserReposition")
```

Comment:
I think so, as written. I haven't traced the code paths, but what if we guard the reposition event for userwindows and the root Geyser container for "main" (which we should maybe refactor as a Geyser.UserWindow against "main"?) to check the height/width for the window they reside in, and only continue with the reposition of itself and its children if the current value for either one is different than the last stored value.

Reply:
I didn't notice any resizing, so that maybe is already happening? Will re-test tomorrow to be sure.

Reply:
I mean if you are manually re-sizing - what does Geyser redraw then?

Reply:
I didn't explain well. Geyser reposition is repositioning everything by moveWindow and resizeWindow but if the value didn't (if I reposition a UserWindow the value for main window won't change) it is not doing anything, at least I didn't notice any change even with big UI.
But to be 100% sure I will change the code to only affect the resized window (because it's a pretty simple change)